content/architecture/infrastructure/deployment/altinn-apps/_index.md

```diff
@@ -18,92 +18,139 @@ The following diagram shows the deployment architecture for Altinn Studio togeth
+Pods, where the Altinn App are delpoyed, gets access to secrets during deployment via HELM charts.
 
-#### Kubernetes Namespace
+### Kubernetes Namespace
 
 It's not planned to use namespaces.
 
-#### Routing
+### Kubernetes Service
+
+Each Altinn App will be a Kubernetes Service.
+
+### Routing
 
-To be able to route traffic to the correct container, each container is tagged to a specific
-end user service. The routing mecahnism routes to the correct container based on the url
-containing the service parameter.
+Routing in Kubernetes is handled by an Ingress-Controller.
 
-### Data services
+To be able to route traffic to the correct Kubernetes Service (Altinn App), each container is tagged to a specific
+Altinn App. The routing mecahnism routes to the correct Kubernetes Service based on the url
+containing the Altinn App parameter.
 
-The data services application is the application responsible exposing data related functionality 
-to the service containers. This container will be scaled based on need.
+## Data services
 
-### Platform Cluster
+The data services application is the application responsible exposing data related functionality
+to the Altinn App. This container will be scaled based on need.
 
-The platform cluster in Altinn Studio Apps will host common application like DataServices. 
+## Platform Cluster
 
-#### Platform integration
+The platform cluster in Altinn Studio Apps will host common application like DataServices.
 
-The platform integration is a new application hosted in the existing infrastructure. 
-It exposes REST-APIs for Profile, Register, Authorization, Intermediary and Authentication. 
-These are services not part of the Altinn Studio Platform (yet) and
-everyone planning to run the Altinn Studio platform would need to implement their own components that support. 
+### Platform integration
 
-### Networking
+The platform integration is a new application hosted in the existing infrastructure.
+It exposes REST-APIs for Profile, Register, Authorization, Intermediary and Authentication.
+These are services not part of the Altinn Platform (yet) and
+everyone planning to run the Altinn Platform would need to implement their own components that support. (?)
 
-#### Sub domains
+## Networking
 
-Each service owner will have their own sub domain.
+### Sub domains
+
+Each organisation will have their own sub domain.
 
 > org.apps.altinn.no
 
-#### Path for the service
+### Path for the Altinn App
 
-> org.apps.altinn.no/servicename
+> org.apps.altinn.no/appname
 
-#### Encrypted traffic and network policy
+### Encrypted traffic and network policy
 
 There is ongoing analysis related to this topic. [Click here to find more information on Github](https://github.com/Altinn/altinn-studio/issues/1000).
 
-#### Ingress / Load balancer
+### Ingress-Controller
 
-Traefik will be used as an Ingree / Load balancer.
+Traefik will be used as an Ingress-Controller.
```

Comment:
cant this be a part of the routing section where Ingress-Controller is already mention?

Reply:
Relocated.